src/components/features/home/contact/contact-form/ContactForm.tsx

```diff
@@ -0,0 +1,79 @@
+import { Form } from 'react-router-dom'
+import './contact-form.scss'
+import { useForm } from 'react-hook-form'
+import * as yup from 'yup'
+import { yupResolver } from '@hookform/resolvers/yup'
+
+const schema = yup.object({
```

Comment:
Pomyślałabym nad jakimś osobnym plikiem i folderem który by miał w sobie takie modele/schematy bo napewno będzie ich troszkę, ale to można przemyśleć i zrobić zmiany jak połączymy kod w dev

Reply:
Ja na ten czas użyłam folder `models` bo miałam enum'a którego właśnie tam umieściłam, ale to jeszcze do przekminy